test/steps/data.go

```diff
@@ -134,6 +135,16 @@ func (data *Data) ResolveWithScenarioContext(str string) string {
 	return result
 }
 
+// ResolveTableWithScenarioContext replaces the last cell of table with the scenario context.
+func (data *Data) ResolveTableWithScenarioContext(table *godog.Table) {
+	if len(data.ScenarioContext) > 0 {
+		for _, row := range table.Rows {
+			cellsCount := len(row.Cells)
+			row.Cells[cellsCount-1].Value = data.ResolveWithScenarioContext(row.Cells[cellsCount-1].Value)
```

Comment:
I think this is an useful method in order to replace the last columns of tables with the scenario context.